lib/ncr_dispatcher.rb

```diff
@@ -13,11 +13,18 @@ def final_approval(proposal)
   end
 
   # Notify approvers who have already approved that this proposal has been
-  # modified
+  # modified. Also notify current approvers that the proposal has been updated
   def on_proposal_update(proposal)
     proposal.approvals.approved.each{|approval|
-      CommunicartMailer.proposal_notification_email(
-        approval.user_email_address, approval, false).deliver
+      CommunicartMailer.notification_for_approver(approval.user_email_address, approval, "already_approved").deliver
+    }
+    proposal.currently_awaiting_approvals.each{|approval|
+      if approval.api_token
```

Comment:
What are you *actually* checking for here? Maybe an `approval.notified?` method or something?